utils/scapp.h

```diff
@@ -414,7 +524,7 @@ scApp::scApp(string& version, string& defaultVersion,
       : ktxApp(version, defaultVersion, options), options(options)
 {
   argparser::option my_option_list[] = {
-      { "bcmp", argparser::option::no_argument, NULL, 'b' },
+      { "encode", argparser::option::required_argument, NULL, 'b' },
```

Comment:
We shouldn't use 'b' for this. Is 'e' being used anywhere here, in ktxApp or in toktx?

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/c01d05d4cfa0353f848c18051d90ba94862115b5 using `o` instead for now.

Reply:
Don't use o as it is used for output in ktxsc and it is documented.

I have not documented most of the short options in usage because I'm not confident they won't have to change. I think it is best to just have the long option, in other words, use a number.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/18af4983456764eb784babe2d11a125f5d2b7f5c